contrib/babelfishpg_tsql/src/session.c

```diff
@@ -51,6 +53,15 @@ get_cur_db_name(void)
 	return pstrdup(current_db_name);
 }
 
+void 
+set_cur_db_name_for_parallelWorker(const char* logical_db_name)
+{
+    int len = strlen(logical_db_name);
+    strncpy(current_db_name, logical_db_name, MAX_BBF_NAMEDATALEND);
+	current_db_name[len] = '\0';
+}
```

Comment:
Why dont we need validation on db_name? There should be Assert on the len <= MAX_BBF_NAMEDATALEND

Reply:
As we were setting this db name using get_cur_db_name() only where we are already checking the length, so IMO it was redundant. But, there is always a possibility for the data to get corrupted. Hence, put a check for length in latest rev.